bus-mapping/src/circuit_input_builder.rs

```diff
@@ -374,7 +376,7 @@ impl CircuitInputBuilder<DynamicCParams> {
             // needed.
             let max_keccak_rows = 0;
             FixedCParams {
-                max_rws: max_rws + 1,
+                max_rws: max_rws + 3,
```

Comment:
Note: A workaround to fix padding overflow
See https://github.com/privacy-scaling-explorations/zkevm-circuits/issues/1507